shard.override.yml

```diff
@@ -0,0 +1,6 @@
+dependencies:
+  retriable:
+    github: Sija/retriable.cr
+  awscr-signer:
```

Comment:
What the reason for the override? Looks like it's a first-party - any reason the different ref can't be listed straight in `shard.yml`?

Reply:
it's a temporary fix, and not an explicit dependency